src/dmd/expression.d

```diff
@@ -1246,7 +1246,7 @@ private:
  */
 extern (C++) int RealEquals(real_t x1, real_t x2)
```

Comment:
Calling this function `Equals` and then doing an `Identity` comparison is vastly confusing. `==` and `is` are fundamentally different operations for floating point, and we need to be careful not to conflate them.

Reply:
#9918